File: Client/Web/ClientApp/app/routes/astro/horoscope.ts
export interface AstroSign {
    name: string;
    order: number;
    technicalName: string;
    start: Date;
    end: Date;
}
export interface HoroscopeTopic {
    title: string;
    totalStars: number;
    stars: number;
}

export interface Horoscope {
    sign: AstroSign;
    globalText: string;
    topics: HoroscopeTopic[];
}

File: Common/Astro/Common.Astro.Model/Horoscope.cs
using System;
using System.Collections.ObjectModel;
using System.Text;

namespace Common.Astro.Model

 
{
	public class Horoscope
	{
		public AstroSign Sign { get; set; }
		public String GlobalText { get; set; }
		public ObservableCollection<HoroscopeTopic> Topics { get; set; }
		public Horoscope()
		{
			Topics = new ObservableCollection<HoroscopeTopic>();
		}

		public override string ToString()
		{
			StringBuilder builder = new StringBuilder();
			builder.AppendLine(Sign.Name);
			builder.AppendLine(GlobalText);
			foreach (HoroscopeTopic horoscopeTopic in Topics)
			{
				builder.AppendLine(horoscopeTopic.ToString());
			}
			return builder.ToString();
		}
	}
}
